### PR TITLE
Revert commit

### PR DESCRIPTION
**Pull request by dcho5**

### CODE REVIEW

**Review by nirzari — CHANGES_REQUESTED** [review]
Please use `git revert` command for reverting the commit. You can read more about it online. e.g., https://www.atlassian.com/git/tutorials/undoing-changes/git-revert

After that if you run into some merge conflicts, please resolve them manually. Make sure you don't alter the lines from the latest copy in https://github.com/dcho5/cachito/blob/master/docker/Dockerfile-workers which are not part of the commit to be reverted. For removing any dependencies added as part of the older commit, please confirm with @chmeliik if they are not being used by anything else in the module.

Finally, please use a single commit to submit your changes. Read online regarding the command `git squash`

**Comment by tkdchen** [comment]
Any context about why revert the commit?

**Comment by chmeliik** [comment]
@tkdchen see CLOUDBLD-3526. On that note, @dcho5 please always include the CLOUDBLD ticket in the commit message when applicable. In fact, a commit message should always have these parts:
```
Short description here

CLOUDBLD-xxxx

[optional: more detailed description]

Signed-off-by: name <email>
```
To add the signed-off-by line, you can use the `-s` option when making the commit. To add the line to an existing commit, you can use `git commit --amend -s`

File: docker/Dockerfile-workers [hunk 1]
```diff
@@ -6,15 +6,13 @@ RUN dnf -y install \
     --setopt=deltarpm=0 \
     --setopt=install_weak_deps=false \
     --setopt=tsflags=nodocs \
-    gcc \
-    gettext \
```
[the remark this answers is not in the document]
Comment (chmeliik) [reply docker/Dockerfile-workers:10]: I think we don't

File: docker/Dockerfile-workers [hunk 2]
```diff
@@ -6,15 +6,13 @@ RUN dnf -y install \
...
-    git-core \
```
[the remark this answers is not in the document]
Comment (chmeliik) [reply docker/Dockerfile-workers:11]: We do need git

**Review by nirzari — APPROVED** [review]

**Comment by nirzari** [comment]
LGTM

**Comment by tkdchen** [comment]
LGTM

**Pull request merged**